bin/jekyll_pages_api

```diff
@@ -0,0 +1,160 @@
+#! /usr/bin/env ruby
+# Author: Mike Bland <michael.bland@gsa.gov>
+# Date:   2015-06-21
+
+require_relative '../lib/jekyll_pages_api'
+
+module JekyllPagesApi
+  class GeneratedSite
+    attr_reader :baseurl, :basedir, :title_prefix, :body_element
+    attr_accessor :pages
+
+    def initialize(baseurl, basedir, title_prefix, body_element)
+      @baseurl = baseurl
+      @basedir = basedir
+      @title_prefix = title_prefix
+      @body_element = body_element
+      @pages = Array.new
+    end
+
+    def each_site_file
+      Dir[File.join self.basedir, '**', '*'].each do |f|
+        next unless f.end_with? '.html'
```

Comment:
I'd imagine so. The gem already parses non-Markdown-generated `.html` files since #24.